### PR TITLE
Treat relative paths as relative to the project root. (#19)

### PR DESCRIPTION
**Pull request by Osmose**
@bwinton r?

### CODE REVIEW

File: lib/advanced-open-file-view.coffee [hunk 1]
```diff
@@ -20,6 +20,19 @@ isRoot = (inputPath) ->
...
 
+absolutify = (inputPath) ->
+  ###
+  Ensure that the given path is absolute. Relative paths are treated as
+  relative to the current project root.
+  ###
+  if getRoot(inputPath) == "."
+    projectPaths = atom.project.getPaths()
+    if projectPaths.length > 0
+      return path.join(projectPaths[0], inputPath)
```
Comment (bwinton) [inline lib/advanced-open-file-view.coffee]: Did we really want to just use the first path, or do we want to be a little cleverer, and try and see which project path matches the most of the input?  (i.e., if I've got a project with 'a/', and another with 'b/', and I try to create 'b/c', I'm betting I meant to create it in the second project root…)

We could also use the project root of the currently open file, if such exists…

File: README.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ Hit `Cmd-Alt-O`/`Ctrl-Alt-O` to open the file list to the directory of the
 current file. As you edit the path the file list will automatically show
 matching files and directories. Hit `Tab` to autocomplete the path.
 
+Relative paths are considered relative to the current project's root folder.
```
Comment (bwinton) [inline README.md]: "relative to the project's _first_ root folder", if we decide not to make the other change…

**Comment by bwinton** [comment]
Other than that, it looks pretty good to me…

File: lib/advanced-open-file-view.coffee (hunk 1 above)
Comment (Osmose) [inline lib/advanced-open-file-view.coffee]: I like the idea of simultaneously using all the project roots. I tried implementing this but it needs better UI to convey to the user just what it's doing. For now, I think I'll stick with the first project's path, and spend some time thinking about what the UI for multiple project support should look like.

**Pull request merged**